src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostBuilderExtensions.cs

```diff
@@ -135,6 +136,26 @@ public static IHostBuilder ConfigureWebJobs(this IHostBuilder builder, Action<Ho
             return builder;
         }
 
+        /// <summary>
+        /// Configures the specified <see cref="IHostBuilder"/> as a scale manager host.
```

Comment:
Is that something a customer will do? Or do we need to mention this is only for internal infrastructure or something like that?

Reply:
yes, this will be called from `ScaleController and a webjobs customer is not suppose to create a "scale manager host". I I extended the description.